sim/druid/item_sets_pve_phase_8.go

```diff
@@ -354,9 +357,26 @@ func (druid *Druid) applyScarletEnclaveGuardian6PBonus() {
 		return
 	}
 
-	core.MakePermanent(druid.RegisterAura(core.Aura{
-		Label: label,
-	}))
+	savageFlurryAura := druid.RegisterAura(core.Aura{
+		ActionID: core.ActionID{SpellID: 1226127},
+		Label:    "Savage Flurry",
+		Duration: time.Second * 10,
+		OnGain: func(aura *core.Aura, sim *core.Simulation) {
+			druid.MultiplyAttackSpeed(sim, 1.3)
+		},
+		OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+			druid.MultiplyAttackSpeed(sim, 1.0/1.3)
+		},
+	})
+
+	core.MakeProcTriggerAura(&druid.Unit, core.ProcTrigger{
+		Name:     label,
+		Callback: core.CallbackOnSpellHitDealt,
+		Outcome:  core.OutcomeCrit,
+		Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
+			savageFlurryAura.Activate(sim)
+		},
+	})
```

Comment:
https://wago.tools/db2/SpellAuraOptions?build=1.15.7.60013&filter%5BSpellID%5D=1226126&page=1
https://wowdev.wiki/Spell.dbc/procFlags

Looking at the proc flag on this, it appears to work on both direct and periodic crits without restrictions. So we probably want to make this a custom aura with both callbacks